ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponentInvokeSql.java

```diff
@@ -0,0 +1,32 @@
+package io.syndesis.qe.pages.integrations.edit;
+
+import static com.codeborne.selenide.Condition.visible;
+
+import org.openqa.selenium.By;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Created by sveres on 12/7/17.
+ */
+@Slf4j
+public class ActionConfigureComponentInvokeSql extends ActionConfigureComponent {
```

Comment:
Maybe a parent class with defined `inputFill` method and then specify them to Invoke and Periodic flavour?

Reply:
you mean abstract inputFill method in parent?

Reply:
Yes, to have parent with all the common code and then Periodic child with one additional method to specify interval.